### DISCUSSION TITLE
Preview relase of the GPU version using SYCL

### DISCUSSION THREAD

**Discussion by Xiangyu-Hu** [Xiangyu-Hu/SPHinXsys#491] [Announcements]
This is a preview release of SPHinXsys in GPU version.
Currently, only the dam-break cases are running on GPU.
Please feel free to fork, try and test the performance.

For installing this version, please follow the instruction from 
https://www.sphinxsys.org/html/installation.html#installing-sycl-version-on-ubuntu

If you have any suggestion or comments, please use the issue and discussion page.
You can also contact me (xiangyu.hu@tum.de) for possible collaborations on the GPU version.

<hr /><em>This discussion was created from the release <a href='https://github.com/Xiangyu-Hu/SPHinXsys/releases/tag/v1.0-beta.08-sycl'>Preview relase of the GPU version using SYCL</a>.</em>

**Comment by DrChiZhang** [comment]
oneAPI DPC++ Compiler documentation
https://intel.github.io/llvm-docs/UsersManual.html
